### ISSUE TITLE
APRs not showing up

### ISSUE THREAD

**Issue by gmbronco** [balancer/backend#527]
Optimism 0x8bb826afc0ff7d2c034a2883f4c461ffd238e1c300020000000000000000012b - should have ~7% from tokens

```
query AprSample {
  poolGetPool(
    chain: OPTIMISM
    id: "0x8bb826afc0ff7d2c034a2883f4c461ffd238e1c300020000000000000000012b"
  ) {
    dynamicData {
      aprItems {
        apr
        __typename
      }
    }
  }
}
```

**Comment by franzns** [comment]
we are still using the deprecated the graph urls, need to find the new ones.

**Comment by gmbronco** [comment]
fixed in #530
